qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/upgrades/FeatureUpgradeIT.java

```diff
@@ -100,7 +100,11 @@ public void testGetFeatureUpgradeStatus() throws Exception {
 
                 assertThat(feature, aMapWithSize(4));
                 assertThat(feature.get("minimum_index_version"), equalTo(getOldClusterIndexVersion().toString()));
-                if (getOldClusterVersion().before(TransportGetFeatureUpgradeStatusAction.NO_UPGRADE_REQUIRED_VERSION)) {
+
+                boolean migrationNeeded = parseLegacyVersion(getOldClusterVersion()).map(
```

Comment:
Worth a comment why this is still using version?

Reply:
Comment added